src/hooks/useLogDownloader/index.ts

```diff
@@ -17,28 +17,35 @@ import {
 import { fetchLogFile } from "utils/fetchLogFile";
```

Comment:
it's not a big deal, but do you know where this flash of 0 comes from? (only happens on bigger logs)

https://github.com/evergreen-ci/parsley/assets/47064971/5a98864c-3cdb-4eb2-babd-c492171cbdb3

Reply:
I actually think this is a React strict-mode quirk that occurs locally and results in certain things firing twice! We've been running into more of those lately. I can reproduce this on `main` when running locally, and don't see the behavior when my branch is deployed to beta.